src/c++/tests/cc_client_test.cc

```diff
@@ -1030,17 +1149,233 @@ TYPED_TEST_P(ClientTest, AsyncInferMultiMismatchOutputs)
   ASSERT_FALSE(err.IsOk()) << "Expect AsyncInferMulti() to fail";
 }
 
+TYPED_TEST_P(ClientTraceTest, UpdateTraceSettings)
```

Comment:
Since the API has subtle difference between HTTP client and Python client, it will be more readable to have separated test suit for HTTP Trace and GRPC Trace (i.e. defining class without template and use TEST_F)

Reply:
Updated, please review.